### ISSUE TITLE
npm ERR! enoent Error while executing

### ISSUE THREAD

**Issue by sebiweise** [w9jds/firebase-action#48]
```
Step 11/15 : RUN npm install -g firebase-tools
 ---> Running in 9add85bc17c0
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/firebase-tools/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -2
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/DABH/diagnostics.git
npm ERR! enoent 
npm ERR! enoent 
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-06-22T06_34_33_145Z-debug.log
The command '/bin/sh -c npm install -g firebase-tools' returned a non-zero code: 254

##[error]Docker build failed with exit code 254
```

Once again. Worked few hours ago.

**Comment by sebiweise** [comment]
#46

**Comment by w9jds** [comment]
It looks like `firebase-tools` has pushed yet another update that requires `git` to be on the container to install it... As this container doesn't have `git` it is failing.

**Comment by w9jds** [comment]
This has been resolved with release v1.5.0
